src/content_script/site_tuner.spec.ts

```diff
@@ -85,7 +85,7 @@ describe('SiteTuner tests', () => {
 
   xit('Tuner handles added comments', (done) => {
     const siteTuner =
-      new SiteTunerImpl(0.5, DEFAULT_ATTRIBUTES, THEMES.dotted);
+      new SiteTunerImpl(0.5, DEFAULT_ATTRIBUTES, THEMES.dotted, false);
```

Comment:
I have a small preference for commenting boolean parameters like this for clarity (false /* subtypesEnabled */) but I'll let you make a call on that.